src/app/main/component/ubs/components/ubs-personal-information/ubs-personal-information.component.ts

```diff
@@ -76,6 +76,11 @@ export class UBSPersonalInformationComponent extends FormBaseComponent implement
       });
   }
 
+  private getLastAddresses(addressList: Address[]) {
+    const lastAddresses = -4;
```

Comment:
what if the user doesn't have 4 addresses ?

Reply:
It will display 3,2,1,0. Depends on what he has